src/mangadex_downloader.py

```diff
@@ -30,45 +29,22 @@ def __init__(self, manga_list_file, export_dir, max_containers, torify):
 
     # Start download container instance
     def start_download(self, instance_name, command_args):
-        if self.torify_it:  # If torify is requested we have to start the container with subprocess
```

Comment:
We're no longer using the torsocks method which didn't actually work, so all this code is unnecessary